org.eclipse.nebula.widgets.nattable.extension.glazedlists.feature/feature.xml

```diff
@@ -19,25 +19,13 @@
       %license
    </license>
 
-   <plugin
-         id="org.eclipse.nebula.widgets.nattable.extension.glazedlists"
-         download-size="0"
-         install-size="0"
-         version="0.0.0"
-         unpack="false"/>
+   <requires>
+      <import plugin="ca.odell.glazedlists"/>
+      <import plugin="org.apache.commons.commons-codec"/>
+   </requires>
```

Comment:
```suggestion
   <requires>
      <import plugin="ca.odell.glazedlists"/>
      <import plugin="org.apache.commons.commons-codec"/>
   </requires>
```

I would assume both are anyways required by some import-package/require bundle so no need to list them here.